src/main/scala/nl.knaw.dans.easy.bagstore/server/DefaultServletComponent.scala

```diff
@@ -18,15 +18,22 @@ package nl.knaw.dans.easy.bagstore.server
 import java.net.URI
 
 import nl.knaw.dans.easy.bagstore.component.BagStoresComponent
-import nl.knaw.dans.easy.bagstore.server.ServletEnhancedLogging._
+import nl.knaw.dans.lib.logging.servlet.masked.MaskedAuthorizationHeader
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import nl.knaw.dans.lib.logging.servlet._
 import org.scalatra.{ Ok, ScalatraServlet }
 
+import scala.language.postfixOps
+
 trait DefaultServletComponent {
   this: BagStoresComponent =>
 
   val defaultServlet: DefaultServlet
 
-  trait DefaultServlet extends ScalatraServlet with ServletEnhancedLogging {
+  trait DefaultServlet extends ScalatraServlet with DebugEnhancedLogging
+    with ServletLogger
+    with MaskedAuthorizationHeader
```

Comment:
is this necessary here? not sure...

Reply:
Both bagsServlet and StoresServlet need an instance of dans.lib.logging.servlet.ServletLogger.
I thought extending from DebugEnhancedLogging would be the best way to go.

Reply:
I meant the `MaskedAuthorizationHeader`. not sure if that one would be necessary here...

Reply:
Yes, because we want to mask the auth token which was logged in plain text while we were testing